auto-instrumentation/okhttp/okhttp-3.0/README.md

```diff
@@ -1,5 +1,7 @@
 # Android Instrumentation for OkHttp version 3.0 and higher
 
+# 🚧Work in progress 🚧
```

Comment:
I added this line because I think we need to warn people not to use this tool until [this issue](https://github.com/open-telemetry/opentelemetry-java/issues/5886) is solved.